tests/unit/onyxCacheTest.tsx

```diff
@@ -540,14 +541,14 @@ describe('Onyx', () => {
                         const key = `${ONYX_KEYS.COLLECTION.MOCK_COLLECTION}${number}`;
                         return {
                             key,
-                            connectionId: Onyx.connect({key, callback: jest.fn()}),
+                            connection: Onyx.connect({key, callback: jest.fn()}),
                         };
                     });
                 })
                 .then(waitForPromisesToResolve)
                 .then(() => {
                     // When a new connection for a safe eviction key happens
-                    Onyx.connect({key: `${ONYX_KEYS.COLLECTION.MOCK_COLLECTION}9`, callback: jest.fn()});
+                    Onyx.connect({key: `${ONYX_KEYS.COLLECTION.MOCK_COLLECTION}10`, callback: jest.fn()});
```

Comment:
Why this change?

Reply:
We generate 10 connections from 0 to 9, and this line is connecting again to the connection 9. Since the connection manager is in place this won't create a new connection, but rather reuse the connection 9 previously created. This part of the test requires a new connection to be made, so I just incremented the number. Another way to solve is:

```ts
Onyx.connect({key: `${ONYX_KEYS.COLLECTION.MOCK_COLLECTION}9`, callback: jest.fn(), reuseConnection: false});
```